migrations/V240__Add_Secondary_Foreign_Key.sql

```diff
@@ -219,11 +224,28 @@ secondary_fkey = (optional) the column name of the column that holds a secondary
   This will track both milestone_id as the primary foreign key and solution_id as the secondary foreign key.
 ';
 
--- Re-configure mto_milestone_solution_link to track solution_id as secondary foreign key
-SELECT audit.AUDIT_TABLE('public', 'mto_milestone_solution_link', 'id', 'milestone_id', '{created_by,created_dts,modified_by,modified_dts}'::TEXT[], '{*,milestone_id}'::TEXT[], 'solution_id');
-
--- Re-configure plan_document_solution_link to track document_id as secondary foreign key
-SELECT audit.AUDIT_TABLE('public', 'plan_document_solution_link', 'id', 'solution_id', '{created_by,created_dts,modified_by,modified_dts}'::TEXT[], '{document_id}'::TEXT[], 'document_id');
-
--- Re-configure existing_model_link to track current_model_plan_id as secondary foreign key
-SELECT audit.AUDIT_TABLE('public', 'existing_model_link', 'id', 'model_plan_id', '{created_by,created_dts,modified_by,modified_dts}'::TEXT[], '{existing_model_id, current_model_plan_id}'::TEXT[], 'current_model_plan_id');
+-- Update mto_milestone_solution_link to track solution_id as secondary foreign key
+UPDATE audit.table_config
```

Comment:
I think any field that we attach a secondary FK field probably needs to have that field included in the insert fields to make sure existing functionality is not broken.